examples/go.mod

```diff
@@ -15,3 +15,4 @@ require (
 )
 
 replace github.com/charmbracelet/bubbletea => ../
+replace github.com/charmbracelet/bubbles => ../../bubbles
```

Comment:
I see why you needed this change, but it will obviously fail on the CI and for most users (who don't have bubbles checked out manually). You could bump the required bubbles to a specific commit id, which contains the textarea changes with:

```
go get github.com/charmbracelet/bubbles@abcdef
```

Reply:
Sorry about that! Remnants of development 😅